lib/Valetudo.js

```diff
@@ -48,6 +49,30 @@ const Valetudo = function() {
         vacuum: this.vacuum,
     });
 
+    if(this.configuration.get("telegram_token")) {
+        this.telegramClient = new Telegrambot({ 
```

Comment:
I believe it should be better to be able to completely disable Telegrambot in config.json for users who never use telegram, so this would never be called for them.

Reply:
Current code creates `Telegrambot` object regardless of `telegram_token` being set or not, that I was talking about. In my country, for example, telegram is forbidden by the law. I can't directly use it. Why should I have its components loaded into valetudo so they would consume limited vacuum's RAM for nothing? There should be an ability to completely disable it when it's not needed.